exporter/util.go

```diff
@@ -34,6 +35,8 @@ import (
 	pconfig "github.com/prometheus/common/config"
 )
 
+var sanitzieValueRE = regexp.MustCompile(`(,|_)`)
```

Comment:
Rather than use a regexp, how about a string replacer?

```suggestion
var sanitzieValue = strings.NewReplacer(",", "", "|", "")
```

Reply:
```diff
- var sanitzieValue = strings.NewReplacer(",", "", "|", "")
+ var sanitzieValue = strings.NewReplacer(",", "", "_", "")
```